aws_health_to_slack/program.py

```diff
@@ -59,7 +58,7 @@ def _aws_health_events() -> list[dict]:
     https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/health/client/describe_events_for_organization.html
     """
     try:
-        mins = int(re.match(r"(\d+)m", os.getenv("TRIGGER_INTERVAL")).group(1))
+        mins = int((os.getenv("TRIGGER_INTERVAL") or "1m")[:-1])
```

Comment:
```suggestion
        mins = int((os.getenv("TRIGGER_INTERVAL") or "1m")[:-1]) # Remove unit suffix and parse as int
```
Not attached to wording, but that wasn't obvious to me so I think a comment helps